README.md

```diff
@@ -21,6 +21,60 @@ Usage:
 - Run `quarto preview` to serve the site in Quarto's local webserver
 - Review the [`_quarto.yml`](_quarto.yml) file and [Github Actions workflow](.github/workflows/gh-pages.yml) for more details
 
+## Generating API reference documentation
+
+The repository includes an API documentation generator that takes documentation
+comments from Autonity protocol contracts and converts them into Markdown files.
+
+**TL/DR:** Symlink an [autonity](https://github.com/autonity/autonity) repository
+clone into the root of this repository and run [`apidocgen`](./_apidocgen/). The
+configuration is in [`config.toml`](./config.toml); the generated files are saved
+into `./contracts/<autonity-tag>/`.
+
+### Setup
+
+1. Clone the Autonity Go Client into an arbitrary directory:
+   ```sh
+   git clone git@github.com:autonity/autonity.git /path/to/autonity
```

Comment:
I'm not sure whether we need such detailed instruction--what level of expertise do we expect the audience to have?